apps/onboarding/scenes/welcome/index.styl

```diff
@@ -23,6 +23,20 @@
     font-size font-size-lg
     margin-top 2em
 
+    &__skip
+      position absolute
+      top 0
+      right 0
+      padding unit unit 40px 40px
+      cursor pointer
+      opacity 0.45
+      z-index rotate_z + 1
```

Comment:
what is this

Reply:
also there's &times; the icon font x is chunky and ridiculous looking

Reply:
copy pasta from modal style

Reply:
please don't do this to me

Reply:
Just trying to have them look the same, not trying to torture you